### ISSUE TITLE
Accessibility: Missing visible keyboard focus

### ISSUE THREAD

**Issue by krallowski** [humhub/wiki#437]
There are some elements which do not have a visible keyboard focus. I hope it's fine if I sum it up in one issue:

- Seperate pages to select in the overview
- Linked profile of the author of a wikipage
- "history / Historie" link in a wikipage

More a general issue also in core

- comment icon
- like icon

### What steps will reproduce the problem?
Tab through the various elements.

### What is the expected result?
A visual indicator where the keyboard focus actual is: maybe underlined or border

### What do you get instead?
No visual indicator

### Additional info
Version: 2.5.10
